docs/Calamari/Node Operators/get_calamari.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 2
+---
+
+# Get Calamari
+
+## Choose your installation variant
+
+### Option 1: Build From Source
+
+Use this option if you are comfortable working with Cargo and Rust source code and have time or a fast multi-core processor for compiling.
+
+The following instructions are for latest Ubuntu distribution.
+
+```bash
+# install build deps
+sudo apt-get update
+sudo apt-get -y install build-essential clang curl git libssl-dev make pkg-config
+
+# install rust toolchain
+curl -s https://sh.rustup.rs -sSf | sh -s -- -y
+source ~/.cargo/env
+rustup toolchain install nightly
+rustup toolchain install stable
+rustup default stable
+rustup target add wasm32-unknown-unknown --toolchain nightly
+rustup update
+command -v wasm-gc || cargo +nightly install --git https://github.com/alexcrichton/wasm-gc --force
+
+cargo install \
+  --git https://github.com/Manta-Network/Manta \
+  --branch manta-pc \
+  --features calamari \
+  --verbose \
+  --locked \
+  --force
+
+OR...
```

Comment:
I'm not sure what step this refers to. Maybe a better organization is:
```
# Get Calamari
## Option 1: Build from source
### 1) Install Rust Toolchain
### 2) Clone Repo
...
## Option 2: Use the Binary Release
### 1) Download Binary
### 2) Download Relay Chain Spec File
...
```

Reply:
Is it clearer now with 1.1, 1.2, 1/3, etc
Basically there are two ways to do 1.1